### PR TITLE
APM-138 Add SBAS disable option for test

### PR DESCRIPTION
**Pull request by rabad-matternet**
- added option to disable SBAS for GPS1, GPS2, or both
- SBAS disabled via RC

### CODE REVIEW

File: libraries/AP_GPS/AP_GPS_UBLOX.cpp [hunk 1]
```diff
@@ -1020,28 +1020,31 @@ AP_GPS_UBLOX::_parse_gps(void)
...
-                if (_buffer.sbas.mode != gps._sbas_mode) {
-                    _buffer.sbas.mode = gps._sbas_mode;
+                if (_buffer.sbas.mode != desired_sbas_mode) {
+                    _buffer.sbas.mode = desired_sbas_mode;
                     _send_message(CLASS_CFG, MSG_CFG_SBAS,
                                   &_buffer.sbas,
                                   sizeof(_buffer.sbas));
                     _unconfigured_messages |= CONFIG_SBAS;
                     _cfg_needs_save = true;
+                    gcs().send_text(MAV_SEVERITY_INFO, "UBLOX setting sbas mode(%d)", desired_sbas_mode);
```
Comment (zalice) [inline libraries/AP_GPS/AP_GPS_UBLOX.cpp]: Could you move this up before `_buffer.sbas.mode = desired_sbas_mode;` on L1033 and print the current mode and desired mode in the gcs message? That will help us easily check whether the mode reset with powercycle

Also could this be MAV_SEVERITY_ALERT?

Comment (rabad-matternet) [reply libraries/AP_GPS/AP_GPS_UBLOX.cpp]: Done!

**Review by viet-nguyen-matternet — APPROVED** [review]

**Review by dat-tran-matternet — APPROVED** [review]

**Review by zalice — APPROVED** [review]

File: libraries/AP_Compass/AP_Compass.cpp [hunk 2]
```diff
@@ -1541,6 +1541,18 @@ Compass::get_healthy_mask() const
     return healthy_mask;
 }
 
+bool
+Compass::healthy(void) const
```
Comment (AydenChubbic) [inline libraries/AP_Compass/AP_Compass.cpp:1545]: Is this notation of return type being placed one line above function name intentional?

**Review by AydenChubbic — APPROVED** [review]
Slacked you some questions about comment notation for function descriptors but looks good!

**Review by dat-tran-matternet — APPROVED** [review]

File: libraries/AP_Compass/AP_Compass.cpp (hunk 2 above)
Comment (zalice) [reply libraries/AP_Compass/AP_Compass.cpp:1545]: Yes because that is the convention of this file

**Pull request merged**